### DISCUSSION TITLE
ZHA developers needed to review PR for Home Assistant's ZHA docs

### DISCUSSION THREAD

**Discussion by Hedda** [zigpy/zigpy#981] [ZHA development]
@Adminiuga @dmulcahey Any chance you could review this PR for ZHA integration documentation?

https://github.com/home-assistant/home-assistant.io/pull/18864

And any chance ZHA (and zigpy) developers could approve the last revision 'as-is' as it is now?

Original PR is from Aug 2021 so would be great if could just get this in now and improve it later?

Hopefully, you agree that the last revision is at least readable enough to work as a starting point.

Yes it could be better but I think it is better than nothing and this is something others can improve.

**Comment by Adminiuga** [comment]
You were told multiple times not to tag people directly or be banned, yet you still do it.

That PR fails multiple checks, nobody is going to merge a PR with the failing checks.
